apps/meteor/tests/end-to-end/api/oauthapps.ts

```diff
@@ -107,6 +107,23 @@ describe('[OAuthApps]', () => {
 		});
 	});
 
+	describe('[/oauth-apps.info]', () => {
```

Comment:
add a case for a user without the `manage-oauth-apps` perm

Reply:
also a case for fetching an invalid oauth app

Reply:
I added the specified cases to the e2e tests for `/oauth-apps.info`